docs/agent-config.md

```diff
@@ -1,25 +1,40 @@
-# Configuration parameters
+# Configuration options
 
 ## NOTE: subject to change!
 
-Note: These parameter names are very likely to change over time, so please check
-back here when trying out a new version! Please report any bugs or unexpected
-behavior you find.
+Note: The environment variables/system properties in this document are very likely to change over time. 
+Please check back here when trying out a new version! 
+
+Please report any bugs or unexpected behavior you find.
 
 ## Contents
 
 * [SDK Autoconfiguration](#sdk-autoconfiguration)
 * [Peer service name](#peer-service-name)
 * [DB statement sanitization](#db-statement-sanitization)
-* [Customizing the OpenTelemetry SDK](#customizing-the-opentelemetry-sdk)
 * [Suppressing specific auto-instrumentation](#suppressing-specific-auto-instrumentation)
 
 ## SDK Autoconfiguration
 
 The SDK's autoconfiguration module is used for basic configuration of the agent. Read the
-[docs](https://github.com/open-telemetry/opentelemetry-java/tree/v1.0.0/sdk-extensions/autoconfigure#customizing-the-opentelemetry-sdk)
+[docs](https://github.com/open-telemetry/opentelemetry-java/tree/v1.0.0/sdk-extensions/autoconfigure)
 to find settings such as configuring export or sampling.
 
+Here are some quick links into those docs for the configuration options for specific portions of the SDK & agent:
+
+* [Exporters](#exporters)
+  + [OTLP exporter (both span and metric exporters)](https://github.com/open-telemetry/opentelemetry-java/tree/v1.0.0/sdk-extensions/autoconfigure/README.md#otlp-exporter-both-span-and-metric-exporters)
```

Comment:
Hmmm...I'm not convinced that linking to the specific version (instead of `main` branch) is a good idea.  Can you convince me?

Reply:
I have no idea...it's what was linked to in this document, so I stuck with it.

Reply:
`main` sounds reasonable to me

Reply:
updated to point at `main`